src/test/java/org/simlar/simlarserver/services/createaccountservice/CreateAccountServiceTest.java

```diff
@@ -330,6 +332,47 @@ public void testCreateAccountRequestTotalLimitWithinOneDay() {
         assertCreateAccountRequestSuccess(telephoneNumber, "192.168.1.23");
     }
 
+    @DirtiesContext
+    @Test
+    public void testCreateAccountRequestTotalLimitWithinOneHourRegionalLimit() {
+        final int max = settingsService.getRegionals().get(0).getMaxRequestsPerHour();
+        for (int i = 0; i < max; i++) {
+            final String telephoneNumber = "+1600502214" + i % 10;
+            final String ip = "192.168.23." + (i % 255 + 1);
+            reset(smsService);
+
+            if ((i & 1) == 0) {
```

Comment:
Wouldn't `i % 1` be more comprehensible?

Reply:
Yes!
I think I did this because of an inspection warning about speed improvements. But it does not trigger anymore. I changed it in the complete project. New Commits are comming.